TrafficSimulator/Assets/Camera/DefaultCamera.cs

```diff
@@ -0,0 +1,122 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cam
+{
+    /// <summary>
+    /// Represents the initial camera state. It provides functionality for moving, rotating,
+    /// and zooming the camera, as well as handling input and toggling between objects.
+    /// It also has a number of serialized fields for configuring movement and zoom speed,
+    /// minimum and maximum zoom, and edge scrolling
+    /// behavior (which is not yet implemented).
+    /// </summary>
+    public class DefaultCamera : CameraState
+    {
+        // Edge Scrolling is not implemented yet!
+        [SerializeField] private bool _enableEdgeScrolling;
```

Comment:
```suggestion
        private bool _enableEdgeScrolling;
```
Do not show in inspector if it is not implemented yet

Reply:
Fixed! Implemented moving the camera via screen edges now! :)